src/System.Device.Gpio/System/Device/Pwm/Channels/UnixPwmChannel.Linux.cs

```diff
@@ -12,6 +12,7 @@ namespace System.Device.Pwm.Channels
     /// </summary>
     internal class UnixPwmChannel : PwmChannel
     {
+        private readonly TimeSpan OpenTimeout = TimeSpan.FromSeconds(1);
```

Comment:
We use cammel-case for variables like this, so it would be _openTimeout instead.